index.d.ts

```diff
@@ -0,0 +1,6 @@
+import {Component} from 'react';
+interface IDocumentTitleProps {
```

Comment:
I would rename the interface `DocumentTitleProps`. It is not recommended to prefix interfaces with `I` (although its somewhat still up for debate).